packages/api/src/command/webhook/webhook.ts

```diff
@@ -6,6 +6,7 @@ import { nanoid } from "nanoid";
 import { v4 as uuidv4 } from "uuid";
 import { Product } from "../../domain/product";
 import WebhookError from "../../errors/webhook";
+import { isNoWebhookPongEnabledForCx } from "../../external/aws/appConfig";
```

Comment:
nit: no enabled === disabled

Reply:
discussed offline - FF is "NoWebhookPong", so this is saying whether that FF is enabled or disabled for the cx

has pros/cons, but it at least semantically consistent with the rest